Unaccountable-Magpie/BangazonSite
# Ready for Review - Added Items under Categories in Index

### Purpose:

Added Items under Categories in Index

### How it fits in context of the project:
(Link to ticket: # )
Ticket https://github.com/Unaccountable-Magpie/BangazonSite/issues/43
### Specific Feature affected:
Product Types Index

### How to test (Be thorough!):

1. git fetch -a
2 .git checkout evTesting
3. start Bangazon.sln
4. Press IIS Express GREEN play button
5.  Click on Product Types
6.  Check if you see the three item categories listed and a number beside the category.
7. Click on an item to see if it shows the details
8. To run unit testing, type dotnet test into the nuget package manager and ensure that the test passes.
